include/nbl/video/IGPUCommandBuffer.h

```diff
@@ -737,14 +737,20 @@ class NBL_API2 IGPUCommandBuffer : public IBackendObject
             if (invalidBufferBinding({range.offset,range.buffer},alignment,usages))
                 return true;
             if ((range.size&(alignment-1)) && range.size!=asset::SBufferRange<IGPUBuffer>::WholeBuffer)
+            {
+                m_logger.log("Size %d not aligned to %d for the command!", system::ILogger::ELL_ERROR, range.size, alignment);
```

Comment:
you want to use a macro like in `ILogicalDevice` so that `__LINE__` `__PRETTY_FUNCTION__` are printed

Reply:
you can undef before the header ends to stop pollution